dev/com.ibm.ws.webcontainer.servlet.3.1/src/com/ibm/ws/webcontainer31/upgrade/H2HandlerImpl.java

```diff
@@ -86,17 +90,10 @@ public void handleRequest(HttpInboundConnection hic, HttpServletRequest request,
             h2uh.init(new H2UpgradeHandler());
         }
         
-        // create a map of the headers to pass to the transport code
-        Map<String, String> headers = new HashMap<String, String>();
-        HttpServletRequest hsrt = (HttpServletRequest) request;
-        Enumeration<String> headerNames = hsrt.getHeaderNames();
+        String http2Settings = ((HttpServletRequest) request).getHeader(HTTP2_SETTINGS);
```

Comment:
@billw1024 can you pls review this section of H2?  I'm not familiar with the H2 flow.